sdk/src/types/client.ts

```diff
@@ -11,9 +13,9 @@ export interface DirectusClient<Schema extends object> {
  * All used globals for the client
  */
 export interface ClientGlobals {
```

Comment:
I'd make this a type instead of an interface but technically it doesnt really matter